firefox-ios/firefox-ios-tests/Tests/XCUITests/ZoomingTests.swift

```diff
@@ -163,6 +162,12 @@ class ZoomingTests: BaseTestCase {
     }
 
     func zoomIn() {
+        if app.buttons.matching(identifier: AccessibilityIdentifiers.ZoomPageBar.zoomPageZoomLevelLabel).count > 1 {
+            // If there are multiple matches for this element, then both the normal tab and the private tab views may be
+            // in the view hierarchy simultaneously. This should not change unintentionally! Check the Debug View Hierarchy.
+            XCTFail("Too many matches! Has the UI hierarchy unexpectedly changed?")
+        }
+
```

Comment:
Ditto for the use of `XCTAssertEqual` here.
```
XCTAssertEqual(pp.buttons.matching(identifier: AccessibilityIdentifiers.ZoomPageBar.zoomPageZoomLevelLabel).count, 1, "Too many matches")
```

Reply:
Sure, I can change both of these! 👍  However, I will need to multiline this specific one, or break the identifier into its own variable to not violate the swiftline line length rule.